packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -1201,24 +1201,25 @@ def call_next_link_request_builder(self, builder: PagingOperationType) -> List[s
         retval: List[str] = []
         query_str = ""
         next_link_str = "next_link"
-        try:
-            api_version_param = next(
-                p
-                for p in self.code_model.client.parameters
-                if p.rest_api_name == "api-version"
-            )
-            retval.append("# make call to next link with the client's api-version")
-            retval.append("_parsed_next_link = urlparse(next_link)")
-            retval.append(
-                "_next_request_params = case_insensitive_dict(parse_qs(_parsed_next_link.query))"
-            )
-            retval.append(
-                f'_next_request_params["api-version"] = {api_version_param.full_client_name}'
-            )
-            query_str = ", params=_next_request_params"
-            next_link_str = "urljoin(next_link, _parsed_next_link.path)"
-        except StopIteration:
-            pass
+        if not self.code_model.options["multiapi"]:
```

Comment:
context: https://github.com/Azure/autorest.python/issues/1485#issue-1390546801
@iscai-msft I think we need to fix it quickly before the vacation otherwise many customers of multiapi package may be broken. This way is not the best way but can keep consistent with before. Please let me know your thoughts.